packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.scss

```diff
@@ -35,6 +35,9 @@ $DropDown-item-height: 32px;
   color: $ms-color-neutralPrimary;
 
   border: none;
+  width: 100%;
+  text-align: left;
```

Comment:
What will happen for RTL configurations?

Reply:
Changing to justify seems to work, though it feels a bit weird to use it.

Reply:
All you need to do to have it work for RTL is to use the sass macro (@include ms-text-align(left), for example)

---
In reply to: [195148171](https://github.com/OfficeDev/office-ui-fabric-react/pull/5150#discussion_r195148171) [](ancestors = 195148171)

Reply:
All you need to do to have it work for RTL is to use the sass macro (@include ms-text-align(left), for example)